frontend/packages/dev-console/integration-tests/features/Knative/eventing-kamletes.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Creation of Kamelets via Event Source Creation
+    As a user, I should be able to install and use more camel connectors
+
+
+    Background:
+        Given user has installed OpenShift Serverless Operator
+        And user has created Knative Serving and Knative Eventing CR
+        And user has installed Red Hat Integration - Camel Operator
+        And user has created Integration Platform CR
```

Comment:
shall we mention namespace here global installation is not there , will see kamelets only in ns we added CR for IP

Reply:
Added the namespace in the background which will run before every scenario.

Reply:
sorry din't got what you mean by background here, can you point to that ??

Reply:
ignore i see this here https://github.com/openshift/console/pull/7441#discussion_r539847326